lib/workers/miq_defaults.rb

```diff
@@ -0,0 +1,26 @@
+require "active_support/core_ext/numeric/time"
+
+module Workers
+  class MiqDefaults
+    HEARTBEAT_TIMEOUT = 2.minutes.freeze
+    STARTING_TIMEOUT  = 10.minutes.freeze
+    STOPPING_TIMEOUT  = 10.minutes.freeze
```

Comment:
Why do we need a new class for these values? Shouldn't we be using the ones in `Settings` [here](https://github.com/ManageIQ/manageiq/blob/d785fa8cfead31cf3fa3a71d280ba2c509df2e3d/config/settings.yml#L1042-L1052).

I'm not sure why they were in two different places to begin with...